lib/common/output_log.c

```diff
@@ -351,3 +375,32 @@ pcmk__output_set_log_level(pcmk__output_t *out, uint8_t log_level) {
     priv = out->priv;
     priv->log_level = log_level;
 }
+
+/*!
+ * \internal
+ * \brief Set the file, function, and line used to filter the next log message
```

Comment:
and tags

Reply:
ah, initially I left tags out but eventually changed my mind